superset/jinja_context.py

```diff
@@ -297,7 +297,7 @@ def get_filters(self, column: str, remove_filter: bool = False) -> List[Filter]:
         filters: List[Filter] = []
 
         for flt in form_data.get("adhoc_filters", []):
-            val: Union[str, List[str]] = flt.get("comparator")
+            val: Union[Any, List[Any]] = flt.get("comparator")
```

Comment:
does this need to be `Any`? Maybe there's a more precise type we can use here? What values are expected for the comparator?

Reply:
If we want to be fully explicit, I believe the full type would be `Union[bool, str, int, float, List[bool], List[str], List[int], List[float]]`, but that feels slightly verbose. Could potentially be shortened by introducing an alias `FilterValue = bool, str, int, float` and then doing `Union[FilterValue, List[FilterValue]]`, but even that feels kinda stuffy. However,   I'd personally vote for keeping it `Any` for now.